spec/controllers/activities_controller_spec.rb

```diff
@@ -134,7 +134,7 @@
         describe 'global' do
           render_views
 
-          before { get 'index', format: 'atom' }
+          before do get 'index', format: 'atom' end
```

Comment:
Prefer {...} over do...end for single-line blocks.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.